src/raster/r.object.thickness/r.object.thickness.py

```diff
@@ -130,18 +196,16 @@ def main():
 
     # check if input file exists
     if not gs.find_file(input)["file"]:
-        gs.fatal(_("Raster map <%s> not found") % input)
+        gs.fatal(_("Raster map {} not found".format(input)))
 
     # strip mapset name
     in_name_strip = options["input"].split("@")
     in_name = in_name_strip[0]
 
-    tmp = str(os.getpid())
-
     # create a map containing only the category to replace and NULL
-    categorymap = "{}".format(in_name) + "_bin_" + "{}".format(tmp)
-    gs.verbose(_("Category map: <%s>") % categorymap)
-    gs.verbose(_("Extracting category: <%d>") % category)
+    categorymap = create_temporary_name(f"{in_name}_bin_")
+    gs.verbose("Category map: {}".format(categorymap))
+    gs.verbose(_("Extracting category: {}".format(category)))
```

Comment:
Same thing here, the only real change done here is removing the translation for Category map

Reply:
Ah, that second line was a mistake. So your changes are the preferable format / form? I can revert my changes, no problem.

Reply:
As said in my other answer, it's only advantageous to use existing translated strings for the time being.